flink-table/flink-sql-client/src/test/resources/sql/set.q

```diff
@@ -118,14 +118,13 @@ SELECT hive_add_one(1);
 Received a total of 1 row
 !ok
 
-# TODO: support this in the FLINK-30692
-#REMOVE JAR '$VAR_UDF_JAR_PATH';
-#[INFO] The specified jar is removed from session classloader.
-#!info
+REMOVE JAR '$VAR_UDF_JAR_PATH';
```

Comment:
Nit: add `SHOW JARS` before and after `REMOVE JAR`?

Reply:
The  show jars statement before the remove jar is at line line 281